frigate/record/cleanup.py

```diff
@@ -18,6 +18,10 @@
 logger = logging.getLogger(__name__)
 
 
+class RecordingsToDelete(Model):
+    id = CharField(null=False, primary_key=False, max_length=30)
```

Comment:
I think it would be good to store this in the models.py file so they are all kept in one place